src/reference/faq.rst

```diff
@@ -70,24 +71,56 @@ The other deciding factor is if you're unable to install Docker (e.g. for admins
 The ambient runtime is not available without WSL.
 
 
+.. old anchors
 .. _why-intel-miniconda-installer-on-apple-silicon:
 
-Why recommend the Intel Miniconda installer for Mac computers with Apple silicon (e.g. M1)?
--------------------------------------------------------------------------------------------
+.. _why-conda-install-errors-on-apple-silicon:
+
+Why do I get an error when installing to my Conda environment on a Mac computer with Apple silicon (e.g. M1)?
+-------------------------------------------------------------------------------------------------------------
```

Comment:
I thought about this some more and came up with https://github.com/nextstrain/bioconda-recipes/issues/3.